sonaranalyzer-dotnet/its/sources/Net5/TargetTypedNew.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Net5
+{
+    public class TargetTypedNew
+    {
+        private Dictionary<string, string> dict = new();
+        private List<int> list;
+
+        public void Method()
+        {
+            list = new();
+            StringBuilder sb = new();
```

Comment:
We can add a test also with constructor parameters (e.g. `StringBuilder sb2 = new("a");`)